spec/factories/minion.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+FactoryGirl.define do
+  factory :minion do
+    sequence(:hostname) { |n| "Minion#{n}" }
```

Comment:
@mssola introduced a similar factory in another PR  (minions_factory.rb). I will use that instead.